### DISCUSSION TITLE
Settings page with multiple sections/forms

### DISCUSSION THREAD

**Discussion by ALSHIHAB** [filamentphp/filament#8896] [Help]
I'm creating a settings resource with multiple sections header, footer, general, slider, and so on.
The table shows each section in a row. 
How to create a form for each row? so that when click edit row it shows the form.
For example, when click edit footer row it opens a footer form to edit.
![setings](https://github.com/filamentphp/filament/assets/43612338/b08c80e8-2b24-45bf-a015-5f13339a61ed)

**Comment by leandrocfe** [comment]
Why not [filament-spatie-settings](https://filamentphp.com/plugins/filament-spatie-settings) ?
